doc/general/operation-modes/operation-modes.md

```diff
@@ -34,6 +34,38 @@ The queue nutshell serves to encapsulate the functionality of a receipt chain fo
 
 The Helper nutshell is used to support updates and configurations.
 
+#### fiskaltrust.SignatureCloud
+
+The fiskaltrust.SignatureCloud is a pure online solution. The receipt linking is handled entirely online. The advantage of this solution is that no installation or configuration is required for the client and any platform can use this service.
+
+The fiskaltrust.SignatureCloud has two different versions:
+ - 1.1
+ - 1.2
+
+We differentiate between them by backend servers. So the external url is the same: https://signaturcloud.fiskaltrust.at for both versions. If you want to use a specific version of fiskaltrust.SignatureCloud, you will need to set the service-version parameter into the header. As default the fiskaltrust.SignatureCloud 1.1 version is used.
```

Comment:
Maybe we should be more specific about http headers here. IMO it makes sense to reference a wiki article or an general information on http headers here.

Reply:
in future version we need to clarify this.

Reply:
is it ok for you if i add the following link (https://en.wikipedia.org/wiki/List_of_HTTP_header_fields) as a reference where you can find more information about the http headers? or i can write a short description about it, as you wish